### ISSUE TITLE
Pico System

### ISSUE THREAD

**Issue by tinspin** [shuichitakano/pico-infones#14]
Hi, did you consider porting this to the Pimoroni Pico System handheld?

**Comment by shuichitakano** [comment]
I don't think it would be difficult to port it, but I don't have that system.
Next time I buy something at Pimoroni, I may try to buy that along with it.
Don't expect too much.

**Comment by tinspin** [comment]
It's a very nice platform, hope you can try it!

**Comment by kefabean** [comment]
I'd also be super interested in a port to picosystem. Unfortunately I wouldn't know where to start, but would be curious to understand what this would take?
